src/mgmt/srv_pool.c

```diff
@@ -287,17 +333,28 @@ ds_mgmt_pool_extend(uuid_t pool_uuid, d_rank_list_t *svc_ranks, d_rank_list_t *r
 		    uint32_t *domains)
 {
 	d_rank_list_t		*unique_add_ranks = NULL;
-	int			ntargets;
-	int			rc;
+	int			 ntargets;
+	daos_prop_t		*prop = NULL;
+	int			 flags;
+	int			 rc;
 
 	D_DEBUG(DB_MGMT, "extend pool "DF_UUID"\n", DP_UUID(pool_uuid));
 
+	rc = get_flags_prop(&prop, pool_uuid, svc_ranks);
+	if (rc != 0) {
+		D_ERROR("creating pool on ranks "DF_UUID" failed:"
+			" pool properties could not be retrieves: "DF_RC"\n",
+			DP_UUID(pool_uuid), DP_RC(rc));
```

Comment:
Using the new logging macros this would be:
```suggestion
		DL_ERROR(rc, "creating pool on ranks "DF_UUID" failed:"
			" pool properties could not be retrieves",
			DP_UUID(pool_uuid));
```

Reply:
- [ ] Replace D_ERROR with DL_ERROR

Reply:
fixed with commit https://github.com/daos-stack/daos/commit/848c314b31e924c42492105a51c687b981900ee8